pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -37,9 +37,9 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
```

Comment:
We must adjust the version number, too.

Reply:
to the convention "version-unstable-date" i imagine ?

Reply:
hmmm, good question, I do not know... maybe the upstream format?